src/core/lifecycle.ts

```diff
@@ -0,0 +1,6 @@
+export interface IChange {
```

Comment:
Maybe lets rename it to `AttributeValueChange`? (with no prefix of 'I', saw it in Angular Material and kinda like it)

Reply:
Okay